hack/non-olm-install/install-gitops-operator.sh

```diff
@@ -394,6 +403,66 @@ function print_info() {
   echo "==========================================="
 }
 
+# migration from an OLM installation to a non OLM installation.
+function migrate_olm_installation() {
+  extract_custom_env_in_subscription
+  remove_subscription
+  remove_installed_csv
+  wait_for_olm_removal
+  if [ -f ${WORK_DIR}/migrate_env.sh ];then
+    echo "Sourcing env variables used for customizing subscription"
+    source ${WORK_DIR}/migrate_env.sh
+  fi
+  apply_kustomize_manifests
+  # Check pod status if it becomes ready
+  check_pod_status_ready gitops-operator-controller-manager
```

Comment:
As discussed offline, we do not really need rollback.. `check_pod_status_ready` calls `rollback` if the pod failed to come up..

Reply:
Implemented the rollback feature for migration. If the migration fails, the script switches back to the OLM installation, so that the error can be corrected and re-run.